compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2187,12 +2187,32 @@ OMR::Z::TreeEvaluator::lbitpermuteEvaluator(TR::Node *node, TR::CodeGenerator *c
    return TR::TreeEvaluator::bitpermuteEvaluator(node, cg);
    }
 
+static TR_ExternalRelocationTargetKind
+getRelocationTargetKindFromSymbol(TR::CodeGenerator* cg, TR::Symbol *sym)
```

Comment:
@Spencer-Comin  I think this change may remove relocations being added by @cjjdespres in https://github.com/eclipse/omr/pull/6956/files#diff-322438948e52d98d89adb41b9fac54130800bf31d82634aa2e9bef9988400f20. Can you make sure that this routine also checks if the symbol is `CatchBlockCounter`  ?